File: auriculoterapia_api/Auriculoterapia.Domain/Paciente.cs
using System;

namespace Auriculoterapia.Domain
{
    public class Paciente
    {   
        public int Id {get; set;}
        public DateTime FechaNacimiento {get; set;}
        public string Celular {get; set;}

        //One to One Relationship
        public int UsuarioId {get; set;}
        public Usuario Usuario {get; set;}

        
    }
}

File: Auriculoterapia/app/src/main/java/com/example/android/auriculoterapia_app/models/Paciente.kt
package com.example.android.auriculoterapia_app.models


import java.io.Serializable



data class Paciente(var id: Int?,
                    var fechaNacimiento: String,
                    var celular: String,
                    var usuario: Usuario): Serializable{

    override fun toString(): String {
        return "Patient(id='$id', fecha='$fechaNacimiento', celular='$celular')"
    }
}